apps/evans/components/checkout/services/CheckoutService.ts

```diff
@@ -2,7 +2,7 @@ import { type T } from '@friends-library/pairql/order';
 import type Cart from '../models/Cart';
 import type { Lang } from '@friends-library/types';
 import type CheckoutApi from './CheckoutApi';
-import { LANG } from '@/lib/env';
+import { LANG } from '../../../lib/env';
```

Comment:
Was this on purpose?

Reply:
yeah, vitest was choking on the import alias. if i find that we have to do this more than once, i think we can teach vitest about the alias, but if this is the only spot causing an issue, i figured it might be better just to use the relative path here than add a bunch of configuration